content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,435 @@
+
+To enable :guilabel:`eLearning` courses on a *Helpdesk* team, go to
+:menuselection:`Helpdesk --> Configuration --> Teams` and select a team, or create a :doc:`new one
+<getting_started>`.
+
+On the team's settings page, scroll to the :guilabel:`Self-Service` section, and check the box next
+to :guilabel:`eLearning`. A new field will appear below, labeled :guilabel:`Courses`.
+
+Click the empty field next to :guilabel:`Courses` (beneath the :guilabel:`eLearning` feature) to
+reveal a drop-down menu. Select an available course from the drop-down, or type a title into the
+field, and click :guilabel:`Create and edit` to create a new course from this page. Multiple
+courses can be assigned to a single team.
+
+Create an eLearning course
+--------------------------
+
+A new :guilabel:`eLearning` course can be created from the :guilabel:`Helpdesk` team's settings
+page (as in the step above), or from the :guilabel:`eLearning` app.
+
+To create a course directly through the :guilabel:`eLearning` application, navigate to
+:menuselection:`Odoo Dashboard --> eLearning --> New`. This reveals a blank course template that
+can be customized and modified as needed.
+
+On the course template page, a :guilabel:`Course Title` and :guilabel:`Tags`.
+
+Click on the :guilabel:`Options` tab. Under :guilabel:`Access Rights`, choose the :guilabel:`Enroll
+Policy`. This determines what users will be allowed to take the course. Under :guilabel:`Display`,
+choose the course :guilabel:`Type` and :guilabel:`Visibility`. The :guilabel:`Visibility` setting
+determines whether the course will be available to public site visitors or members.
+
+Add content to an eLearning course
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add content to a course, click the :guilabel:`Content` tab and select :guilabel:`Add Content`.
+Choose the :guilabel:`Content Type` from the drop-down and upload the file (or past the link) where
+instructed. Click :guilabel:`Save` when finished. Click :guilabel:`Add Section` to organize the
+course in sections.
+
+.. image:: help_center/help-center-elearning-course-contents-page.png
+   :align: center
+   :alt: View of a course being published for Odoo Helpdesk.
+
+.. note::
+   In order to add a certification to a course, go to :menuselection:`eLearning --> Configuration
+   --> Settings` and check the box labeled :guilabel:`Certifications`, to activate the setting.
+
+.. seealso::
+   `Odoo Tutorials: eLearning <https://www.odoo.com/slides/elearning-56>`_
+
+Publish an eLearning course
+---------------------------
+
+To allow customers to enroll in a course, both the course and the contents need to be published.
+
+If the course is published, but the contents of the course are not published, customers can enroll
+in the course, but they won't be able to view an of the course content. Knowing this, it may be
+beneficial to publish the course first if the course contents are intended to be released over time
+(such as classes with a weekly schedule).
```

Comment:
```suggestion
If the course is published, but the contents of the course are not published, customers can enroll
in the course, but they won't be able to view an of the course content. Knowing this, it may be
beneficial to publish the course first if the course contents are intended to be released over time
(such as classes with a weekly schedule).
```

Maybe specify where they enroll for the course